elastic/apm-server
# fix: clear scroll after completing scroll requests

## Motivation/summary

clear the search context using a clear scroll request for sourcemap and agentcfg search scroll requests

## Checklist




For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**?
- Is its use being published in **telemetry** to enable product improvement?
- Have system tests been added to avoid regression?

## How to test these changes

- update a lot of sourcemaps to ES
- run apm-server
- check open scroll context
- verify there are no open scroll context

## Related issues
Related to https://github.com/elastic/apm-server/issues/14619
